docs/resources/agent.md

```diff
@@ -0,0 +1,135 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "nobl9_agent Resource - terraform-provider-nobl9"
+subcategory: ""
+description: |-
+  Agent configuration documentation https://docs.nobl9.com/the-nobl9-agent
```

Comment:
please remove this file before agents are enabled on prod

Reply:
`resource_agent.go` is already present on main branch and so this is only an auto generated file that translates the contents to a doc page... Do you think it's worth removing even though the resource already exists? 

If my reasoning is correct, then I see it could somewhat beneficial... But I'm not sure so I wanted to double check 😸 We should be aware that it'll be naturally generated each time we run `go generate`.

Reply:
Never mind, I see it was explicitly removed some time ago, and so "this is the way".